spec/dummy/app/helpers/application_helper.rb

```diff
@@ -8,4 +8,8 @@ def embedded_svg filename, options={}
     end
     doc.to_html.html_safe
   end
+
+  def hide_published_at(post)
+    @post.is_published ? '' : 'display:none;'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.